pyvo/dal/tests/test_adhoc.py

```diff
@@ -116,7 +116,7 @@ class TestClass(dict, AxisParamMixin):
         test_obj.band.add(())
     with pytest.raises(ValueError):
         test_obj.band.add((1, 2, 3))
-    with pytest.raises(ValueError):
+    with pytest.raises(TypeError):
```

Comment:
This is a slight change in API if anyone is currently relying on `ValueError` for validation. But `TypeError` is more appropriate.